Marlin/src/lcd/dogm/status_screen_DOGM.h

```diff
@@ -468,7 +468,7 @@ static void lcd_impl_status_screen_0() {
   // Status line
   //
 
-  #define STATUS_BASELINE (55 + INFO_FONT_HEIGHT)
+  #define STATUS_BASELINE (53 + INFO_FONT_HEIGHT)
```

Comment:
plus font height… of course…

Reply:
Maybe it should just be fixed, or better, `(64 - INFO_FONT_DESCENT)` … which would then need us to define the descent for each font.

Reply:
That would also work - does the current + INFO_FONT_HEIGHT not include low details? Hence the missing pixel at the bottom?
I am assuming the Font Height is 10? (Can't check as not on my printer)
So 64 - INFO_FONT_HEIGHT would be ample, and the bottom of the text would touch the bottom of the screen - I went for 53 as it looked incomplete still at 54.

Reply:
The original is doing 55 +  INFO_FONT_HEIGHT which if 10 would make it 65 (Missing the screen)
My edit made is 53 + INFO_FONT_HEIGHT which would make it 63 (Leaving a line of pixels at the bottom empty)
So if you made it just 64 as a fixed point it would make all font sizes (if people change fonts?) stick to the bottom of the screen?
Thoughts?